src/chainparams.cpp

```diff
@@ -99,6 +99,9 @@ class CMainParams : public CChainParams {
         // The best chain should have at least this much work.
         consensus.nMinimumChainWork = uint256S("0x0000000000000000000000000000000000000000002cb971dd56d1c583c20f90");
 
+        // By default assume that the signatures in ancestors of this block are valid.
+        consensus.defaultAssumeValid = uint256S("0x0000000000000000030abc968e1bd635736e880b946085c93152969b9a81a6e2"); //447235
```

Comment:
@rebroad This is possible, please read the docs. Also, if you are unsure about the block hash, you can always set -noassumevalid